spec.md

```diff
@@ -4117,11 +4123,12 @@ s.pop()  # error: empty set
 
 `S.remove(x)` removes the value `x` from the set `S`. It returns `None`.
 
-`remove` fails if the set does not contain `x`, is frozen, or has active
-iterators. If you do not want to fail on an attempt to remove a non-present
-element, use [`discard`](#set·discard) instead. If you need to remove multiple
-elements from a set, see [`difference_update`](#set·difference_update) or the
-[`-=`](#sets) augmented assignment operation.
+`remove` fails if the set does not contain `x` (in particular, if `x` is
```

Comment:
Nit: Do you mean "in particular, even if `x` is unhashable"? Without the "even" I'm not sure what "in particular" is emphasizing.

Reply:
I put the "in particular" to indicate that "`x` is unhashable" is a potentially non-obvious subcase of "the set doesn't contain `x`"

Reply:
Reworded to be less confusing.